test/SourceKit/CodeComplete/complete_sort_order.swift

```diff
@@ -9,23 +9,30 @@ func test() {
 
 // XFAIL: broken_std_regex
 // RUN: %sourcekitd-test -req=complete -req-opts=hidelowpriority=0 -pos=7:1 %s -- %s > %t.orig
-// RUN: %FileCheck -check-prefix=NAME %s < %t.orig
-// Make sure the order is as below, foo(Int) should come before foo(String).
+// RUN: %sourcekitd-test -req=complete -req-opts=hidelowpriority=0,sort.byname=0 -pos=7:1 %s -- %s > %t.orig.off
+// RUN: %FileCheck -check-prefix=NAME_SORTED %s < %t.orig
+// RUN: %FileCheck -check-prefix=NAME_UNSORTED %s < %t.orig.off
```

Comment:
How about adding a `diff` check for `%t.orig` vs `%t.orig.off` to ensure we got different behavior?
I suspect the test will still pass even if both invocations end up sorting the results.

Reply:
> I suspect the test will still pass even if both invocations end up sorting the results.

Right.

I will add the `diff` test.